frontend/src/components/ConsultantRow.tsx

```diff
@@ -496,16 +497,18 @@ function DetailedBookingCell({
 }) {
   const [hours, setHours] = useState(detailedBookingHours.hours);
   const [cellId, setCellId] = useState(detailedBookingHours.id);
+  const [isChangingHours, setIsChangingHours] = useState(false);
   const { setIsDisabledHotkeys } = useContext(FilteredContext);
   const router = useRouter();
 
   const organisationName = usePathname().split("/")[1];
+  const numWeeks = detailedBooking.hours.length;
 
-  function updateHours() {
+  function updateHours(hourlyChange?: number) {
     setIsDisabledHotkeys(false);
     setDetailedBookingHours(
```

Comment:
a function with 10 params..? 
I assume this is older code, but could we clean it ut using an object instead?

Reply:
Btw. Would it be a suggestion to enforce a maximum numbers of params a function can have? E.g this:
https://typescript-eslint.io/rules/max-params/

Reply:
I just created a PR with a fix for this, so no worries 😄